carla/cf_models/api/cfmodel.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC, abstractmethod
+
+
+class CFModel(ABC):
```

Comment:
General thing, should we name the it *Methods* instead of CFModel? To get some clarity with the naming, also since we are going to have recourse methods?

The readme also referenced it as such

> CARLA is a python library to benchmark counterfactual explanation and recourse models. It comes out-of-the box with commonly used datasets and various machine learning models. Designed with extensibility in mind: Easily include your own counterfactual methods, new machine learning models or other datasets.

Reply:
Good idea, maybe naming it _Recourse_Methods_ would make it clear for every user.